install_config/topics/vsphere_prerequisites.adoc

```diff
@@ -155,3 +155,14 @@ If {product-title} node VMs are created from a virtual machine template, then
 you can set `disk.EnableUUID=1` on the template VM. VMs cloned from this
 template inherit this property.
 ====
+
+[[vsphere-vmotion]]
+== Using {product-title} with vMotion
+[IMPORTANT]
+====
+{product-title} generally supports compute-only vMotion. Storage vMotion is not supported.
+====
+
+If you are using vSphere volumes in your pods, migrating a VM across datastores either manually or through Storage vMotion causes invalid references within {product-title} persistent volume (PV) objects. This prevents affected pods from starting up and can result in data loss. Therefore, any form of Storage vMotion is unsupported.
```

Comment:
```suggestion
If you are using vSphere volumes in your pods, migrating a VM across datastores either manually or through Storage vMotion causes invalid references within {product-title} persistent volume (PV) objects. These references prevent affected pods from starting up and can result in data loss. Therefore, any form of Storage vMotion is unsupported.
```

Reply:
Also made this change in the 4.x PR ([reflink](https://github.com/openshift/openshift-docs/pull/29499/files#diff-a2fa1e73a5f56b4ce57913aa8e5137fb38ae101ee14240333fb0abcf401dbcd3R52)).

Reply:
Yes, I made the change too!